src/applications/claims-status/containers/ClaimsStatusApp.jsx

```diff
@@ -122,12 +121,13 @@ AppContent.propTypes = {
 };
 
 function ClaimsStatusApp({ dispatchSetLastPage, featureFlagsLoading, user }) {
-  const history = useHistory();
```

Comment:
Updated ClaimStatusApp.jsx to use useLocation() instead of useHistory() since useHistory() was removed with react-router V6